README.md

```diff
@@ -46,7 +46,7 @@ GamePlay Video
 We strive to launch a project with one button. But now there are a few manual steps that must be performed before the game can be launched from the editor.
 
 1. **Copy scheme catalog from `/Zilon.Client/Assets/Resources/Schemes` into `bin/Content`**.
-The game required the scheme catalog in its `bin` directory. The easiest way to place the scheme catalog is executing `copy_scheme_catalog_to_game_bin.bat` from root of the repository.
+The game required the scheme catalog in its `bin` directory. The easiest way to place the scheme catalog is executing `copy_scheme_catalog_to_game_bin.ps` from root of the repository.
```

Comment:
The extension of PowerShell scripts should be `ps1`. Not just `ps` (https://en.wikipedia.org/wiki/PowerShell).

Can it runs on Linux?